system/translations.rst

```diff
@@ -1,80 +1,116 @@
 Translations
 ************
 
-Translations of Zammad are processed centrally on our
+Translations of Zammad are processed centrally in our
 `Weblate instance <https://translations.zammad.org/>`_.
-This allows non development people to review and update translation strings of
-the language they actually speak.
+This allows non-developers to review and update translation texts of the
+language they actually speak.
 
-Your language is in an incomplete translation state or has typos?
-Please consider helping us sorting this out! All you need is either a GitHub
-account or register directly on our instance.
+Your language is in an incomplete translation state or has typos? Please
+consider helping us in sorting this out! All you need is either a GitHub account
+or to register directly in our instance.
 
-Did you know? This is also where documentation translations are handled. 🤓
+Did you know? This is also where the documentation translations are handled. 🤓
 
 Have a look in our
 :docs:`contribution section </contributing/start.html#translation>` in the
 system documentation to get started!
 
 --------------------------------------------------------------------------------
```

Comment:
Just a minor note: I tried to avoid the separation lines and even deleted some of them. Perhaps it can be structured by a proper heading (e.g. "Introduction" and "Local Translation" or something like that)? But feel free to ignore it ;-)

Otherwise it looks good!!